spec/adapters/postmark_adapter/outbound_spec.rb

```diff
@@ -74,6 +74,14 @@
 
         subject { message_email[:from].formatted }
         it { should eq(['TestingProject <100eyes-test-account@example.org>']) }
+
+        context 'with a comma / list separator in the project name' do
+          before do
+            allow(Setting).to receive(:project_name).and_return('TestingProject, with a comma!')
+          end
```

Comment:
Not sure if the line break was mandated by rubocop, but in general I prefer the `{ … } ` block syntax for single-line blocks.

```suggestion
          before { allow(Setting).to receive(:project_name).and_return('TestingProject, with a comma!') }
```